tests/phpDocumentor/Reflection/DocBlock/ContextFactoryTest.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * This file is part of phpDocumentor.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @copyright 2010-2015 Mike van Riel<mike@phpdoc.org>
+ * @license   http://www.opensource.org/licenses/mit-license.php MIT
+ * @link      http://phpdoc.org
+ */
+
+namespace phpDocumentor\Reflection\DocBlock {
+
+// Added imports on purpose as mock for the unit tests, please do not remove.
+    use Mockery as m;
+    use phpDocumentor\Reflection\DocBlock,
+        phpDocumentor\Reflection\DocBlock\Tag;
```

Comment:
Please read the comment above ;)

This is done on purpose because the test file is used as mock